ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/autovpn-edge.cfg

```diff
@@ -26,23 +26,52 @@ router path-selection
          name autovpn-rr2
          ipv4 address 10.8.8.8
    !
-   load-balance policy LBPOLICY
+   load-balance policy LB-CONTROL-PLANE-PROFILE
       path-group INET
    !
-   policy dps-policy-default
+   load-balance policy LB-DEFAULT-AVT-POLICY_it
+      path-group INET priority 2
```

Comment:
Is this valid? Having no priority 1 path-group?

Reply:
it is accepted by CLI